src/actions/pipelines.js

```diff
@@ -136,8 +135,7 @@ export function loadPipelineData(pipelineID) {
         active: pipelineID,
       });
 
-      const shouldExpandAllPipelines =
-        display.expandAllPipelines || expandAllPipelines;
+      const shouldExpandAllPipelines = expandAllPipelines;
```

Comment:
we do not need this assignment if we are getting rid of display.expandAllPipelines. We can just use `expandAllPipelines` from above